lib/dragonfly/data_storage/cloudinary_store.rb

```diff
@@ -25,6 +25,26 @@ def url_for(uid, options = {})
         ::Cloudinary::Utils.cloudinary_url(public_id(uid), options)
       end
 
+      # Store the data AND meta, and return a unique string uid
+      def write(content, opts={})
+        store(content, opts)
+      end
+
+      # Retrieve the data and meta as a 2-item array
+      def read(uid)
+        data = retrieve(uid)
+        resource_data = ::Cloudinary::Api.resource(public_id(uid), exif: true)
+        if data
+          [
+              data,     # can be a String, File, Pathname, Tempfile
+              resource_data['exif']      # the same meta Hash that was stored with write
```

Comment:
Line is too long. [88/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.